app/views/passengers/_filters_and_key.html.haml

```diff
@@ -9,9 +9,10 @@
               label: filter.capitalize,
               checked: @filter == filter,
               inline: true
-      = f.submit 'Print'
+      = f.submit 'Print', formaction: passengers_path(format: :pdf)
       -if @current_user.admin?
         %input.btn.btn-secondary#copybtn{type: 'button', value: 'Copy Emails'}
+        = f.submit 'Export CSV', formaction: passengers_path(format: :csv), class: 'btn btn-secondary'
```

Comment:
We've got the `bootstrap_form_tag` helper here in this app, it takes care of the `btn-` classes unless your're doing something "different".

```suggestion
        = f.submit 'Export CSV', formaction: passengers_path(format: :csv)
```